applications/app/crosswords/CrosswordPage.scala

```diff
@@ -31,3 +31,84 @@ class CrosswordPage(val crossword: CrosswordData, content: contentapi.Content) e
     crossword.dimensions.rows * (CellSize + BorderSize) + BorderSize
   )
 }
+
+class CrosswordSearchPage extends MetaData {
+  override def id: String = "crosswords/search"
+
+  override def section: String = "crosswords"
+
+  override def analyticsName: String = "Crosswords search"
+
+  override def webTitle: String = analyticsName
+
+  val year = new DateTime().getYear()
+  val searchYears = 1999 to year
+
+  val crosswordTypes = Seq(
+    "quick",
+    "cryptic",
+    "prize",
+    "quiptic",
+    "genius",
+    "speedy",
+    "everyman"
+  )
+
+  val setters = Seq(
```

Comment:
Since these are constants maybe put them in the `object` - here you create them every time you create one of these objects.